packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -115,8 +124,8 @@ contract SystemConfig is OwnableUpgradeable, ISemver, IGasToken {
     event ConfigUpdate(uint256 indexed version, UpdateType indexed updateType, bytes data);
 
     /// @notice Semantic version.
-    /// @custom:semver 2.2.0
-    string public constant version = "2.2.0";
+    /// @custom:semver 2.3.0
+    string public constant version = "2.3.0";
```

Comment:
Since this is the first contract change towards the future `op-contracts/v1.6.0` release, what do you think about versioning this as `2.3.0-beta.1`? I plan to write up a design doc for a new way to do contract semvers, which basically will be:
- `X.Y.Z-beta.n` during development
- `X.Y.Z-rc.n` once it becomes a release candidate
- `X.Y.Z` once it becomes an official release

The proposal would also support appending `+feature` for compatibility with https://github.com/ethereum-optimism/design-docs/pull/8

So in this case specifically, assuming the SystemConfig will have no breaking changes before the next release, this PR becomes 2.3.0–beta.1,the next PR becomes 2.3.0-beta.2, and we continue incrementing in that fashion until we get to the release candidate version

Reply:
I am in favor of this. I would like to update the table in https://github.com/ethereum-optimism/optimism/issues/10592 to reflect this scheme. Can update the version to be this

Reply:
I updated the table to be broken out to list of the versions of specific contracts we want to release and then a changelog sort of section for each contract

Reply:
https://github.com/ethereum-optimism/optimism/issues/10592 looks really great, thanks a lot for setting that up